src/core/platform_installer.mjs

```diff
@@ -280,7 +274,51 @@ export class PlatformInstaller {
       rendererOptions: {
         collapseSubtasks: false
       }
+    })
+  }
+
+  /**
+   * Return a list of task to copy the node keys to the staging directory
+   *
+   * It assumes the staging directory has the following files and resources:
+   *   ${staging}/keys/s-<nodeId>.key: signing key for a node
+   *   ${staging}/keys/s-<nodeId>.crt: signing cert for a node
+   *   ${staging}/keys/a-<nodeId>.key: agreement key for a node
+   *   ${staging}/keys/a-<nodeId>.crt: agreement cert for a node
```

Comment:
should not be *-public-node?.pem and *-private-node?.pem.

Reply:
updated, good catch, was copy/paste where I moved it from its original location